### PR TITLE
Add Volume Controller

### PR DESCRIPTION
**Pull request by DiegoxK**
This pull request addresses issue #4 by adding a volume control feature to the extension. Users can now adjust the volume of the voice lines played by the extension.

### CODE REVIEW

**Comment by 89netraM** [comment]
First off, thank you for these PRs! This comment addresses both #13 and #14.

The code updates seem fine at first glance, however before taking a closer look I would like to ask you to fix the formating. Notice that a lot of extra lines has been changed (see the [diff](https://github.com/89netraM/hotheaded-vscode/pull/13/files)) due to whitespace changes.  
I would also like you to change the defaults to align with the current behaviour, set default volume to 1.0, and default delay to exactly 0 ms.

**Comment by DiegoxK** [comment]
Got it, i forgot to deactivate the default usage of prettier and i that formatted all the file sorry.
Let me work on it

**Comment by DiegoxK** [comment]
Okay, I believe this branch now only includes the necessary changes made to `Player.ts` and `package.json`, along with a default volume of 1.

File: src/Player.ts [hunk 1]
```diff
@@ -13,7 +15,14 @@ function play(file: string): void {
...
 	});
-	fs.createReadStream(file).pipe(speaker);
+	const v = new Volume();
+  const voiceVolume = workspace
+    .getConfiguration("hotheadedVSCode")
+    .get<number>("voiceVolume") as number;
+  v.setVolume(voiceVolume);
+
+  v.pipe(speaker);
+  fs.createReadStream(file).pipe(v);
```
Comment (89netraM) [inline src/Player.ts:25]: Line 18 has one tab as indentation, as the rest of the file, but 19-25 has two spaces.

File: src/Player.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
+import { workspace } from "vscode";
 import * as Speaker from "speaker";
 import * as fs from "fs";
+import Volume = require("pcm-volume");
```
Comment (89netraM) [inline src/Player.ts:4]: Is this the correct syntax here?

**Review by 89netraM — CHANGES_REQUESTED** [review]
Have you tested these changes?

Can you also commit the `package-lock.json` file. This will help when building in the future.

File: src/Player.ts (hunk 2 above)
Comment (DiegoxK) [reply src/Player.ts:4]: I don't really know why a regular import doesn't work with pcm-volume but this is the only way i manage to make it work

Comment (89netraM) [reply src/Player.ts:4]: [The docs](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) suggest it has something to do with how the exports are done, but (`pcm-volume`)[https://github.com/reneraab/pcm-volume/blob/2aaaf05c2e1b69e7419af42ed45e1b6a4b191cce/index.js#L44] seem to do a regular `module.exports`. Weird 👍

**Comment by DiegoxK** [comment]
I have tested the changes, and setting both the maximum and minimum values to 0 produces the default behavior where the voices can be heard for each error

File: src/Player.ts (hunk 2 above)
Comment (DiegoxK) [reply src/Player.ts:4]: Good to know, thanks!

**Review by 89netraM — APPROVED** [review]
Great! Because I can't get it to install on my PC right now (some npm issue), but I'll trust you that it works 👍

Great work! Thank you!

**Pull request merged**